presto-main/src/main/java/com/facebook/presto/cost/CapDistinctValuesCountToTypeDomainRangeLength.java

```diff
@@ -46,12 +46,8 @@ public PlanNodeStatsEstimate normalize(PlanNode node, PlanNodeStatsEstimate esti
             estimate = estimate.mapSymbolColumnStatistics(
                     symbol,
                     symbolStatsEstimate -> symbolStatsEstimate.mapDistinctValuesCount(
-                            distinctValuesCount -> {
-                                if (!isNaN(distinctValuesCount)) {
-                                    return min(distinctValuesCount, domainLength);
-                                }
-                                return distinctValuesCount;
-                            }));
+                            distinctValuesCount -> min(distinctValuesCount, domainLength)
```

Comment:
The semantics did not change here.
```
if (!isNaN(distinctValuesCount)) {
   return min(distinctValuesCount, domainLength);
}
```
is executed if `distinctValuesCount` is KNOWN. So `min()` will return a KNOWN result.

If `disctinctValuesCount` is UNKNOWN it will be returned via `return distinctValuesCount;`